dev-python/python-axolotl-curve25519/python-axolotl-curve25519-0.4.1_p2.ebuild

```diff
@@ -0,0 +1,27 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python2_7 python3_{4,5,6} )
+
+inherit distutils-r1
+
+MY_PV="${PV/_p/-}"
+
+DESCRIPTION="A python wrapper for the curve25519 library with ed25519 signatures"
+HOMEPAGE="https://github.com/tgalal/python-axolotl-curve25519"
+SRC_URI="https://github.com/tgalal/${PN}/archive/${MY_PV}.tar.gz -> ${PN}-${MY_PV}.tar.gz"
+
+LICENSE="BSD GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="test"
+
+DEPEND="dev-python/setuptools[${PYTHON_USEDEP}]"
+
+S="${WORKDIR}/${PN}-${MY_PV}"
+
+python_test() {
+	esetup.py test
```

Comment:
```
Ran 0 tests in 0.000s

OK

Reply:
I tend to drop the tests for `python-axolotl-curve25519`. Yes, I saw that thing with 0 tests, but let it for the case in. There are 2 tests under `python-axolotl-curve25519-0.4.1-2/curve/ed25519/tests`, but it seems, they are incomplete, as they can't run compile standalone, missing the main. When looking into `libsignal-protocol-c`, from this got ported, this seems only be a small port of the test suite, which can't be alone this way. At least, I think so, I am not an expert here.